### ISSUE TITLE
Document Capture: Using Web Components to export a MFE remote app of a framework that is different than the MFE host app

### ISSUE THREAD

**Issue by pandabuilt** [nrwl/nx#9816]
### Documentation issue

- [ ] Reporting a typo
- [ ] Reporting a documentation bug
- [X] Documentation improvement
- [ ] Documentation feedback

### Is there a specific documentation page you are reporting?

Enter the URL or documentation section here.
`https://nx.dev/examples/nx-examples ` is a great document for two different apps. However, they are publish on two urls and don't share a host. 

`https://www.angulararchitects.io/en/aktuelles/multi-framework-and-version-micro-frontends-with-module-federation-the-good-the-bad-the-ugly/` - I read that but I'm having a trouble time translating that into a MFE.config.js / NX app

### Additional context or description
I understand that some people think this is a bad idea. I understand that some people think it's a bad idea that are from NX. However, there were multiple questions on slack regarding this - and I think it's relevant to what people want.

**Comment by Coly010** [comment]
Hi, thanks for raising the issue! 🙂 

We'll be updating our docs very soon.

It will highlight the messaging I provided previously that we believe it is poor practice to mix micro frontends of different tech stacks.

I'll leave some relevant links here:

https://www.thoughtworks.com/radar/techniques/micro-frontend-anarchy
https://twitter.com/swyx/status/1481734940165558272
https://twitter.com/swyx/status/1481727316372131845
https://twitter.com/housecor/status/1139504822930092033

And I'll update this issue when the docs have been updated. However, in the meantime I will be closing this as the intent of this issue is to go deeper down a path we do not agree with for Micro Frontends. 

Thanks again for raising this!!
